src/app/components/algorithms/algorithm-view/algorithm-view.component.ts

```diff
@@ -293,12 +309,66 @@ export class AlgorithmViewComponent implements OnInit, OnDestroy {
       );
   }
 
-  createBreadcrumbHeader(): string {
-    const header = this.algorithm.name;
+  createBreadcrumbHeader(algorithm: AlgorithmDto): string {
+    const header = algorithm.name;
+
+    return algorithm.acronym ? header + ' (' + algorithm.acronym + ')' : header;
+  }
+
+  getRevision(revision: EntityModelRevisionDto): void {
+    this.algorithmService
+      .getAlgorithmRevision({
+        algorithmId: this.algorithm.id,
+        revisionId: revision.id,
+      })
+      .subscribe(
+        (algorithmRevision) => {
+          this.frontendAlgorithm = algorithmRevision;
+          let subheading = this.frontendAlgorithm.computationModel
```

Comment:
this code here is exaclty the same as in https://github.com/UST-QuAntiL/qc-atlas-ui/pull/123/files#diff-5bfc3b3483dc91598e30219e090f18123a4d464dbb63836c21d1d7eac1948cb0R290
I'd recommend refactoring it into a utility method.

Reply:
The mentioned code has been refactored as suggested